### ISSUE TITLE
Weird Terrain/Biome Generation.

### ISSUE THREAD

**Issue by Zarepheth** [Chuunkyz/BenchmarkII-1.18#24]
So, my first world with this modpack, created with modpack version 0.4.5, and my spawn area is near a plains biome where I saw a hill with exposed stone nearby.

However, I thought it strange that the hill also exposed blocks of ice and snow! I thought "Plains" was a temperate biome (medium temperature, medium or dry moisture). There was no surface snow - but as I'm tunneling into this hill as my initial base, I'm coming across pockets of ice (which frequently turn into water source blocks) and pockets of snow - both regular and powder. Anyway, I don't consider snow a good building material. The ice either drops nothing, or turns to water (depending upon type and side from which broken). So I'm seriously short on cobblestone for beginning game items and constructing a base with a bit more blast protection than snow and dirt.

- Is this normal for Minecraft 1.18?
- Is this due to modpack settings?
- Is this due to specific mods in the pack, or their settings?
- Did I just get terribly unlucky?

World Settings:
- Bonus Chest: True
- Seed: 1426095319952528101 (randomly selected)
- World type: default
- Generate Features: true

Notes:
- I had to install Java 17 to get Minecraft to launch. I think that should be listed as a requirement versus a recommendation.
- I've progressed far enough to work around the cobblestone shortage, but the pockets of ice, snow, and crevices filled with water are still a challenge when expanding my base.

**Comment by Chuunkyz** [comment]
the only mod doing worldgen here is wild world, i think you're just unlucky

**Comment by Zarepheth** [comment]
So, I created a new world with the current modpack release (0.4.8), using the same seed. The same and terrain is present - although, the generation of snow, ice, rock types, and ores varies from my first world. I'm guessing there is separate generation for those items, or the modpack changes caused those items to generate differently.

I think my spawn point and bonus chest were in the exact same place. However, the plants generated differently.

Continuing my investigations, I created a new world with a randomly selected seed (`7426508238284180395`). This time, I spawned in a Tiaga biome, with swamp, forest, and windswept hills nearby. In every biome where I could peer into caves, crevices, and other openings into the earth, I saw patches of ice and snow where I would have expected nothing but various types of stone.

As a further test, I did a new Vanilla 1.18.1 world (seed: `1547647197967420895`). Although I came across some snow/ice capped hills - I did not see any snow or ice underground. I'll look further into the biome/world gen mods to see if anyone else is seeing what I'm seeing. I assume those mods are Traverse and Wild World. Any others?

**Comment by Zarepheth** [comment]
I did some more experimenting.

Disabling "WildWorld" from the CurseForge modpack profile prevents the addition of underground Ice and Snow.
Surprisingly enough, disabling that mod did not change the ground/air/cave generation -- at least in the small area where I spawned and explored.

I don't know what, if any, config options exist for Wild World, but if the right options exist, it might be possible to reduce or even remove the nonsensical ice and snow patches in regular stone. (I'm fine with ice and snow in a "Frozen Cave" underground biome.)
